lib/optimove_flutter.dart

```diff
@@ -195,6 +198,19 @@ class Optimove {
   }
 }
 
+class OptimoveInAppButtonPress {
+  final Map<String, dynamic>? deepLinkData;
+  final int messageId;
+  final Map<String, dynamic>? messageData;
+
+  OptimoveInAppButtonPress(this.deepLinkData, this.messageId, this.messageData);
+
+  OptimoveInAppButtonPress.fromMap(Map<String, dynamic> map)
+      : deepLinkData = map['deepLinkData'] != null ? Map<String, dynamic>.from(map['deepLinkData']) : null,
```

Comment:
deepLinkData is not optional here?

Reply:
Shouldn't be null, just safer to check

Reply:
1. isn't it more like hiding a potential bug? It comes from base SDKs where the field is not optional, so, it would crash before anyway if smth was wrong with in-app. However, if it was not added by mistake in native proxy part -- this could get missed.
2. consumers of the SDK have access to `OptimoveInAppButtonPress`, and deepLinkData there should be not optional. Arguably getting a deep link without data is useless

Reply:
The only problem is that in case there is some mistake in the native channel part, this code will crash.
Although I am not sure this is a core feature and it worth crashing the app for, changed this to be non optional for now.